store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10555,4 +10555,10 @@ TODO: delete them permanently from here
   <defaultCOSValue>TRUE</defaultCOSValue>
   <desc>Feature to enable/disable the mobile sync for shared folders. Default value is TRUE. The option to sync the shared folders to the Mobile will be enabled for the users in the webclient. The option will only be enabled for shared folders having Admin or Manager permission</desc>
 </attr>
-</attrs>
+
+<attr id="4135" name="zimbraFeatureDeliveryStatusNotificationEnabled" type="boolean" cardinality="single" optionalIn="cos,account" flags="accountInherited,accountInfo" since="10.1.4">
+  <defaultCOSValue>FALSE</defaultCOSValue>
+  <desc>Zimbra feature to send delivery notification when sending email</desc>
```

Comment:
"Zimbra" is unnecessary. Please check descriptions of existing attributes and follow the manner.
For example,
`Whether to enable send delivery notification when sending email`
`enable send delivery notification feature when sending email`

I'm not sure if "when sending email" is necessary or not.
If a short explanation is necessary, add `When true, [how it works]` or something like that.